### PR TITLE
Half the docker image size. Show qrencode usage in docs.

### PR DESCRIPTION
**Pull request by ksylvan**
Through some layer-reducing magic and judicious use of alpine package tools, I reduced the image size by half:

```
$ docker image ls
REPOSITORY             TAG                 IMAGE ID            CREATED             SIZE
new                    latest              ff196a3c00ee        23 minutes ago      90 MB
kayvan/vipaccess       latest              924c623acd94        15 hours ago        198 MB
```

See https://blog.replicated.com/engineering/refactoring-a-dockerfile-for-image-size/
and http://www.sandtable.com/reduce-docker-image-sizes-using-alpine/

Also added a quick mention of how to use the codes generated with TOTP apps like Authy.

### CODE REVIEW

**Comment by ksylvan** [comment]
@dlenski Wait a bit before merging this. I'm trying to improve it so `vipacces show` will work correctly using the image too.

**Comment by ksylvan** [comment]
Okay. This now works nicely. Also updated the docs to show the easiest way to use the docker image. So, no need to install python and packages, just run the docker image.

File: README.md [hunk 1]
```diff
@@ -60,27 +60,30 @@ If you have Docker installed, you can simply use the
...
+Alternatively, you can build it:
 
 1. Check out this repository by running
    ``git clone https://github.com/dlenski/python-vipaccess.git``
 2. Switch to the ``python-vipaccess`` directory by running
    ``cd python-vipaccess``
 3. Install the ``vipaccess`` module
 
-   -  With [pip](https://en.wikipedia.org/wiki/Pip_(package_manager)): ``pip install .``
+   -  With [pip](https://en.wikipedia.org/wiki/Pip_(package_manager): ``pip install .``
```
Comment (dlenski) [inline README.md]: The second `)` needs to stay here, otherwise the link breaks.

**Comment by dlenski** [comment]
What was wrong with `vipaccess show`?

File: README.md (hunk 1 above)
Comment (ksylvan) [inline README.md]: Duh. Fixing.

**Comment by ksylvan** [comment]
Without an existing (empty) `/root/.vipaccess`, the following command:

``docker run --rm kayvan/vipaccess show -s SECRET``

produced:

```
usage: vipaccess show [-h] [-s SECRET | -f DOTFILE | -v]
vipaccess show: error: argument -f/--dotfile: path does not exist: '/root/.vipaccess'
```

Now, with the `.vipaccess` there, it does the right thing.

File: README.md (hunk 1 above)
Comment (ksylvan) [inline README.md]: Fixed.

**Comment by ksylvan** [comment]
My reading of the python code for `show` seems like the intent is to be able to bypass the existence check of the `~/.vipaccess` file:

```python
def show(p, args):
    if args.secret:
        secret = args.secret
    else:
        with open(args.dotfile, "r") as dotfile:
            d = dict( l.strip().split(None, 1) for l in dotfile )
```

but the arg parser for `show` deals with the dot file like this:

```python
    m.add_argument('-s', '--secret',
                   help="Specify the token secret on the command line (base32 encoded)")
    m.add_argument('-f', '--dotfile', type=PathType(exists=True), default=os.path.expanduser('~/.vipaccess'),
                   help="File in which the credential is stored (default ~/.vipaccess")
```

I'm not sure what the "right" way to deal with this is, but that's a different issue than what this PR deals with, which is to provide a working docker image for the tool.

**Comment by ksylvan** [comment]
Anything else you want to be changed?

**Pull request merged**